src/components/header/Header.js

```diff
@@ -36,6 +36,9 @@ function Header() {
           <li>
             <a href="#contact">Contact Me</a>
           </li>
+          <li>
+            <a href="#twitter">Twitter Feed</a>
+          </li>
```

Comment:
I don't think Twitter updates needs it's own place in the menu since if we wish to extend this app to have access to more socials and other features, not every section should get it's own nav as it would make it more cluttered and less minimal which is against the design aesthetic. What do you think @saadpasta ?

Reply:
agree

Reply:
Then can you remove this @naveen521kk . Thanks!